AppInspector/rules/default/cryptography/key_derivation.json

```diff
@@ -42,7 +49,7 @@
     "description": "Cryptography: Key Derivation (PBKDF2)",
     "tags": [
       "Cryptography.KeyDerivation.PBKDF2",
-      "Cryptography.HashAlgorithm.PBKDF1"
+      "Cryptography.HashAlgorithm.PBKDF2"
```

Comment:
I think you can remove this line -- PBKDF2 is only a KDF. But PBKDF1 is also a thing (https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.passwordderivebytes?view=netcore-3.1 and https://github.com/tomyun/crypto-js/blob/d5ac13743fc6af51c2d4bb77986ba045176be092/build/components/pbkdf1.js)